insecure/corruptlibp2p/gossipsub_spammer.go

```diff
@@ -68,7 +67,7 @@ func NewGossipSubRouterSpammerWithRpcInspector(t *testing.T,
 // ctlMessages is the list of spam messages to send to the victim node.
 func (s *GossipSubRouterSpammer) SpamControlMessage(t *testing.T, victim p2p.LibP2PNode, ctlMessages []pb.ControlMessage, msgs ...*pb.Message) {
 	for _, ctlMessage := range ctlMessages {
-		require.True(t, s.router.Get().SendControl(victim.ID(), &ctlMessage, msgs...))
+		s.router.Get().SendControl(victim.ID(), &ctlMessage, msgs...)
```

Comment:
the only difference between the mainline and forked version of pubsub was that in the fork `SendControl` returned a boolean. Since that was only used in this test, I'm removing the check to simplify out dependency management.